RELEASE.md

```diff
@@ -14,12 +14,45 @@
     * Limitador version the operator enables installations of (without prefix) – i.e. `0.X.Y`
     * If the release is a prerelease
 
-3. Verify that the build [release tag workflow](https://github.com/Kuadrant/dns-operator/actions/workflows/build-images-for-tag-release.yaml) is triggered and completes for the new tag
+3. Verify that the build [release tag workflow](https://github.com/Kuadrant/limitador-operator/actions/workflows/build-images-for-tag-release.yaml) is triggered and completes for the new tag
 
 4. Verify the new version can be installed from the catalog image, see [Verify OLM Deployment](#verify-olm-deployment)
 
 5. Release to the [community operator index catalogs](#community-operator-index-catalogs).
 
+### Verify OLM Deployment
+
+1. Deploy the OLM catalog image following the [Deploy the operator using OLM](/doc/development.md#deploy-the-operator-using-olm) and providing the generated catalog image. For example:
+```sh
+make deploy-catalog CATALOG_IMG=quay.io/kuadrant/limitador-operator-catalog:v0.13.0
```

Comment:
Could we add a step here to update the channel to `alpha` given the mismatch with the subscription and the catalog?

Reply:
I will fix this and  manually patch the v0.13.0 catalog to have `stable`

Reply:
done, fixed and patched manually the already existing catalog to have the operator in `stable` channel instead of `alpha`